lib/credo/check/warning/test_exs.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Credo.Check.Warning.TestExs do
+  use Credo.Check,
+    base_priority: :high,
+    param_defaults: [excluded_paths: []],
```

Comment:
As mentioned in a sibling comment, Credo 1.5 allows you to specify which files to run your check on:

```suggestion
    param_defaults: [included: ["test/**/*_test.ex"]],
```

Reply:
This parameter seems to only add additional includes. All files from the project are still sent to the check. This is intentional, right? So I still need to check the source_file against my regex?